internal/oci/runtime_vm.go

```diff
@@ -119,9 +119,10 @@ func (r *runtimeVM) CreateContainer(c *Container, cgroupParent string) (err erro
 
 	defer func() {
 		if err != nil {
-			r.Lock()
-			delete(r.ctrs, c.ID())
-			r.Unlock()
+			logrus.WithError(err).Warnf("Cleaning up container %v", c.ID())
```

Comment:
```suggestion
			logrus.WithError(err).Warnf("Cleaning up container %s", c.ID())
```

Reply:
Harshal, you're reviewing a previous version of the series.

Please, take a look in the latest commits.

Reply:
Ah, no, my mistake, sorry!

Reply:
Changed! Thanks for the review!